### ISSUE TITLE
Transformers.jl failing to load in Julia 1.8 on Mac M1 

### ISSUE THREAD

**Issue by rgreilly** [chengchingwen/Transformers.jl#122]
Hi,

First, many thanks for all your hardwork in setting up and supporting this repository.

I recently tried to run one of the tutorials and had a problem loading and pre-compiling Transformers.jl.

```
ERROR: LoadError: could not load symbol "jl_cpuidex":
dlsym(RTLD_DEFAULT, jl_cpuidex): symbol not found
Stacktrace:
  [1] has_bmi2()
    @ DoubleArrayTries ~/.julia/packages/DoubleArrayTries/3dFUo/src/succinct.jl:13
  [2] top-level scope
    @ none:1
  [3] eval(m::Module, e::Any)
    @ Core ./boot.jl:368
  [4] var"@static"(__source__::LineNumberNode, __module__::Module, ex::Any)
    @ Base ./osutils.jl:19
  [5] include(mod::Module, _path::String)
    @ Base ./Base.jl:419
  [6] include(x::String)
    @ DoubleArrayTries ~/.julia/packages/DoubleArrayTries/3dFUo/src/DoubleArrayTries.jl:1
  [7] top-level scope
    @ ~/.julia/packages/DoubleArrayTries/3dFUo/src/DoubleArrayTries.jl:10
  [8] include
    @ ./Base.jl:419 [inlined]
  [9] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::String)
    @ Base ./loading.jl:1554
 [10] top-level scope
    @ stdin:1
```

I'm using Julia 1.8.2 on a Mac Mini running macOS 13.0.1 on M1 hardware.

Many thanks for your help.

Ronan

**Comment by chengchingwen** [comment]
Hi, I don't have a M1 to test, but the error should be fixed in the new release (v0.1.25). Please update the package and try again.

**Comment by rgreilly** [comment]
Hi,

The new version has fixed the problem.  Many thanks for the prompt solution.

Ronan
